app/Models/Core/Race.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Models\Core;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Race extends Model
+{
+    protected $table = 'races';
+
+    public $timestamps = false;
+}
```

Comment:
Not really sure where to comment this but be aware that races applied to player character's provide certain attributes whereas a monster's race is literally just a string. We can probably just do something like:

````
if ($character->is_player) {
    applyRaceBonus();
}
````

Reply:
Yes, I did say in the PR comment this was like bare min for creating monsters.

Reply:
Can probs link to modifier table and apply the bonus that way?